source/crud/tests/server_2_6/read/aggregate.yml

```diff
@@ -0,0 +1,28 @@
+data:
+    - {_id: 1, x: 11}
+    - {_id: 2, x: 22}
+    - {_id: 3, x: 33}
+serverVersion: '2.6'
```

Comment:
If I understand the [Ruby test runner](https://github.com/mongodb/mongo-ruby-driver/pull/830/files#diff-bc55a144dc56945f9deb7ddec280b86bR5) correctly, this is only going to be executed against a 2.6 server. Shouldn't this test execute for anything >= 2.6?